landlab/components/stream_power/cfuncs.pyx

```diff
@@ -4,6 +4,9 @@ cimport cython
 
 #from libc.math cimport fabs
 
+# as best as KRB can tell, the function _brentq is in c.
+# not sure what, if anything I need to do to 
+from scipy.optimize._zeros import _brentq as brentq
 
```

Comment:
The cython can be seen at https://github.com/scipy/scipy/tree/v0.14.0/scipy/optimize/Zeros; note that when this is implemented in scipy there are a few checks in pure Python that wrap the C code: https://github.com/scipy/scipy/blob/v0.14.0/scipy/optimize/zeros.py#L307

@mcflugen probably has the best handle on how this should work. An import of the _zeros._brentq cython function direct would seem to be the way to go, but beyond my technical skill.